src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -472,6 +504,12 @@ message JobDescriptor {
   // (Optional) Networking configuration. If not defined, sane defaults are
   // provided by the backend.
   NetworkConfiguration networkConfiguration = 11;
+
+  // (Optional) Extra Containers can be specificed to run alongside the main
+  // container in a "pod" (similar to k8s pods). Additional containers
+  // can be specified here, and they will be launched to gether with the main
+  // container, sharing its resources (network/ram/cpu/gpu/etc).
+  repeated BasicContainer extraContainers = 12;
```

Comment:
Seems potentially useful to explicitly call out any ordering (or lack of) assumptions between the basic containers and main container.

Reply:
I'll add that into https://github.com/Netflix/titus-api-definitions/pull/138